.github/workflows/ci.yml

```
@@ -104,6 +105,7 @@ jobs:
     name: Crypto Tests
     runs-on: ubuntu-latest
     needs: build
+    timeout-minutes: 90
```

```suggestion
    timeout-minutes: 30
```

Last run was <2 min, maybe we could lower this to 30?